core/base/supported.go

```diff
@@ -393,24 +390,6 @@ var centosSeries = map[SeriesName]seriesVersion{
 		Version:      "7",
 		Supported:    true,
 	},
-	Centos8: {
```

Comment:
I am confused. We are removing Centos 8 & 9 but not 7 ?

Reply:
Yes. CentOS 7 is still supported by 8 and 9 are EoL

Reply:
Centos9 is a rolling update, so can't be EOL.